azure-communication-ui/demo-app/src/calling/java/com/azure/android/communication/ui/callingcompositedemoapp/SettingsActivity.kt

```diff
@@ -404,6 +404,8 @@ const val MIC_ON_BY_DEFAULT_KEY = "MIC_ON_BY_DEFAULT_KEY"
 const val DEFAULT_MIC_ON_BY_DEFAULT_VALUE = false
 const val CAMERA_ON_BY_DEFAULT_KEY = "CAMERA_ON_BY_DEFAULT_KEY"
 const val DEFAULT_CAMERA_ON_BY_DEFAULT_VALUE = false
+const val CALL_DIAGNOSTICS_ON_BY_DEFAULT_KEY = "CALL_DIAGNOSTICS_ON_BY_DEFAULT_KEY"
```

Comment:
can you please add login in demo app to enable disable before starting call? like other features